app/controllers/declaration/AddPersonController.scala

```diff
@@ -66,10 +74,25 @@ class AddPersonController @Inject () (
     }
   }
 
-  private def isResponsiblePerson(data: AddPerson): Boolean = {
+  def updateFormErrors(f: InvalidForm, businessType: Option[BusinessType]): InvalidForm = {
+    val message = businessType match {
+      case Some(BusinessType.LimitedCompany) => Messages("error.required.declaration.add.position.for.limitedcompany")
```

Comment:
The text `error.required.declaration.add.position.for` is shared as a prefix. It may be worth setting this as a variable.